bin/wpc_init

```diff
@@ -47,3 +61,57 @@ services:
 END
 
 perl -pi -e 's/!!!PROJECTNAME!!!/'${REPLACEMENT}'/g' docker-compose.yml
+
+creating bin/wp <<'END'
+#!/bin/sh
+set -e
+
+FLAGS=
+
+# Add -t flag iff STDIN is a TTY
+if [ -t 0 ]; then
+  FLAGS=-t
+fi
+
+CONTAINER=`docker-compose ps -q wordpress`
+
+# We can't use docker-compose here because `docker-compose exec` is equivalent
+# to `docker exec -ti` and `docker-compose exec -T` is equivalent to
+# `docker exec`. There is no docker-compose equivalent to `docker exec -i`.
+#
+# Issue: https://github.com/docker/compose/issues/3352
+
+docker exec -i ${FLAGS} ${CONTAINER} wp ${@}
+END
+
+creating bin/console <<'END'
+#!/bin/sh
+set -e
+
+exec docker-compose exec wordpress bash
+END
+
+creating bin/setup <<'END'
+#!/bin/sh
+set -e
+
+docker-compose up -d
+`dirname $0`/../setup/external.sh
+docker-compose exec wordpress /usr/src/app/setup/internal.sh
```

Comment:
I'm finding this doesn't always work first time through, depending on how long `external.sh` takes to run. The mysql container takes quite a while to get fully running and connected, so that can result in the `internal.sh` script erroring out because it can't connect to the db. But if you run `setup.sh` again straight afterwards, it works.

See: https://github.com/docker-library/mysql/issues/81